pallets/asset/src/lib.rs

```diff
@@ -1609,39 +1609,15 @@ impl<T: Trait> Module<T> {
             return Ok(ERC1400_TRANSFERS_HALTED);
         }
 
-        if !Identity::<T>::verify_iu_claims_for_transfer(
-            *ticker,
-            to_portfolio.did,
-            from_portfolio.did,
-        ) {
+        if Self::missing_scope_claim(ticker, to_portfolio.did, from_portfolio.did) {
```

Comment:
Function is always called with `portfolio.did`, so let's take the portfolio ids instead